src/Components/CreateImageEntry.jsx

```diff
@@ -30,21 +32,55 @@ class CreateImageEntry extends Component {
     })
   }
 
+  geolocationDataAddress = () => {
+    Geocode.setApiKey(process.env.REACT_APP_API_KEY_GOOGLE_MAPS)
+    Geocode.fromAddress(this.state.userInputAddress).then(
+      response => {
+        const { lat, lng } = response.results[0].geometry.location;
+        this.setState({
+          latitude: lat,
+          longitude: lng
+
+        })
+        this.geolocationDataCoords()
+      },
+      error => {
+        console.error(error);
+      }
+    )
+  }
+
+  geolocationDataCoords = () => {
+    Geocode.setApiKey(process.env.REACT_APP_API_KEY_GOOGLE_MAPS)
+    Geocode.fromLatLng(this.state.latitude, this.state.longitude).then(
+      response => {
+        const addressGeocode = response.results[0].formatted_address
+        this.setState({
+          address: addressGeocode,
+          addressSearch: false
+        })
+      },
+      error => {
+        console.error(error);
```

Comment:
I think an error handler would prove useful.

Reply:
will be done in another chore PR